public/sass/components/_dashboard_grid.scss

```diff
@@ -1,5 +1,5 @@
-@import 'react-grid-layout/css/styles.css';
-@import 'react-resizable/css/styles.css';
+@import 'react-grid-layout/css/styles';
```

Comment:
Removing the `.css` from these imports allows sass to treat them as sass imports and bundle the css into the built sass. Previously these were being compiled to [css imports](https://sass-lang.com/documentation/at-rules/import#plain-css-imports) and caused storybook to fail to compile the sass as it couldn't resolve the css files.